ui/otctrades.ts

```diff
@@ -1,34 +1,50 @@
-var settings = require("./settings.js")();
-require("./elements.js")();
-require("./utils.js")();
+import { settings } from './settings';
+import { dt_edit_drawcallback, showError, showInfo, timestamp_to_date } from './utils';
+import { form_button, form_entry, form_select, form_text  } from './elements';
+import client from './zerorpc_client';
 
-var OTC_TRADES_TABLE = null;
-var CURRENT_TRADE = {};
 
-function update_valhtml(selector_text, val) {
+let OTC_TRADES_TABLE = null as DataTables.Api;
+
+interface TradeData {
```

Comment:
Since I am just starting to understand typescript, what does the interface here do? Why not a class? It's just defining what is inside `CURRENT_TRADE` ?

Reply:
Yes, It just defines the type CURRENT_TRADE should conform to.